### ISSUE TITLE
Feature Request - more than 100 detections

### ISSUE THREAD

**Issue by mattryles** [tensorflow/models#3026]
Hi,

When running the inference the output is only ever 100 num_detections. 

Would it be possible to make the max configurable?

Edit - If it's not possible, as a workarounds would either of these be possible:
1.) Split my label map into 4 and run the inference 4 times with each part of the quarter?
2.) Train 4 separate models with a quarter of the object classes each?

**Comment by michaelisard** [comment]
Which model are you using specifically?

**Comment by mattryles** [comment]
Thanks for the response, Faster RCNN at the moment.

**Comment by mattryles** [comment]
I think I have found the culprit, I'll retrain and go from there.

second_stage_post_processing {
      batch_non_max_suppression {
        score_threshold: 0.0
        iou_threshold: 0.6
        max_detections_per_class: 100
        max_total_detections: 100
      }
      score_converter: SOFTMAX
    }

**Comment by zacwellmer** [comment]
you may also need to lower the iou_threshold if you expect heavy overlapping

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
